README.md

```diff
@@ -45,56 +46,54 @@ events from the device like alerts, removal/insertion events. It then publishes
 | PSME | |  PSME  | | Redfish |
 +------+ +--------+ +---------+
 ```
-The Edgecore Device Manager could manage a wide range of devices (ex: Edgecore XGS-PON/GPON vOLTs, ONL/SONIC Switches, and OpenBMC device).
-Before you use the Device Manager, you have to install the PSME software to the device.
-More detailed information can be found at [DM-Redfish-PSME](https://github.com/opencomputeproject/DM-Redfish-PSME) and [DM-Redfish-OpenBMC](https://github.com/opencomputeproject/DM-Redfish-OpenBMC).
 
-# Preparation
+The device can implement the Redfish Service on an out-of-band platform entity (e.g. BMC) or on an in-band entity (e.g software agent).
+
+The Redfish Service should be conformant to the OCP Baseline Profile.
+The [OCP Baseline profile](https://github.com/opencomputeproject/HWMgmt-OCP-Profiles) prescribes the required [Redfish interface](http://dmtf.org/redfish) support.
+
+The Linux Foundation OpenBMC repository contains source for a BMC firmware image with is conformant to the OCP Baseline profile.
+
+The [OCP DM-PSME repository](https://github.com/opencomputeproject/DM-Redfish-PSME) contains an software agent for a Linux OS.  The image has been used by Edgecore on their XGS-PON/GPON vOLTs and ONL/SONIC switches.
+
+# Hardware and Software requirments
 The Device Manager supports running the *Ubuntu Desktop 16.04 version*. And the hardware platform needs to match the minimum requirement,
 such as *CPU 2 cores/Memory 4GB and SSD driver 40GB*. The host system need to install necessory packages (ex: git, curl and docker)
 
-# Quickly Installation Procedures
-## Install Device Manager using the One-Time-Finish command
-The One-Time-Finish command is installing Device Manager when you build the Device Manager environment first time in the
-Ubuntu 16/18/20 Desktop/Servcer platform. This command will install the necessary packages and commands. It also will build Device Manager containers.
+# Using the Makefile on Ubuntu
 
-```shell
-#> make all
-```
+The following procedures work in Ubuntu 16/18/20 Desktop/Server environment.
 
-## Reinstall Device Manager using the One-Time-Finish command
-The One-Time-Finish command is reinstalling Device Manager after you have installed the Device Manager the first time.
-This command does not install packages and commands. Because you have installed already with those commands before.
-(If the device manager platform has rebooted, you need to perform this command again)
+## Build and Install Device Manager
+The following command will: 1) build the Device Manager containers, 2) install Device Manager and 3) install the packages and commands. After the Once the command completes, Device Manager is ready manage the devices.
```

Comment:
After the Once the command -> After the command

Reply:
Fixed.